app/src/main/res/values/strings.xml

```diff
@@ -104,8 +104,11 @@
   <string name="numeric_input_hint">Enter a number.</string>
   <string name="number_with_units_input_hint_text">Write numbers with units here.</string>
   <string name="numeric_expression_default_hint_text">Type an expression here, using only numbers.</string>
+  <string name="numeric_expression_error_empty_input">Enter a numeric expression to continue.</string>
   <string name="algebraic_expression_default_hint_text">Type an expression here.</string>
+  <string name="algebraic_expression_error_empty_input">Enter an algebraic expression to continue.</string>
   <string name="math_equation_default_hint_text">Type an equation here.</string>
+  <string name="math_expression_error_empty_input">Enter an equation to continue.</string>
```

Comment:
Should be ``math_equation_error_empty_input`` for parity.

Reply:
I will modify, thank you.